src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -532,6 +532,16 @@ static function ( $result ) {
 	// phpcs:ignore PSR2.Methods.MethodDeclaration.Underscore -- Whitelisting to provide backward compatibility to classes possibly extending this class.
 	protected function _list( $_, $assoc_args ) {
 
+		// If `--force-check` and `--skip-update-check` flags are both present, abort.
+		if ( true === (bool) Utils\get_flag_value( $assoc_args, 'force-check', false ) && true === (bool) Utils\get_flag_value( $assoc_args, 'skip-update-check', false ) ) {
+			WP_CLI::error( "{$this->item_type} updates cannot be both force-checked and skipped. Choose one." );
```

Comment:
We generally keep messages starting with capital letter. You can use `ucfirst()` here. Other than this PR looks ok to me.